steps/pvs_like_case_study/implementations/pvs_like_r/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM continuumio/miniconda3
+RUN mkdir -p /input_data
+RUN mkdir -p /results
+VOLUME /results
+VOLUME /input_data
+COPY pvs_like_case_study_r_lock.txt pvs_like_case_study_sample_data_r.py renv.lock ./
+RUN conda init bash
+
+RUN conda create -n pvs_like_case_study_r --file=pvs_like_case_study_r_lock.txt
+RUN echo "conda activate pvs_like_case_study_r" >> ~/.bashrc
+SHELL ["/bin/bash", "--login", "-c"]
+CMD ["bash", "-c", \
+    "Rscript -e 'renv::restore(library=.libPaths())' \
```

Comment:
This should be running at build time, like the `conda create`. It downloads the R packages from the internet and installs them.

Reply:
Perhaps this is why the /var/tmp is filling up as well?

Reply:
The trouble with this is that R needs to be installed. R gets installed by the conda environment, so I just need to figure out how to activate and run that command in a the newly created environment.

Reply:
`RUN conda activate pvs_like_case_study_r && Rscript -e 'renv::restore(library=.libPaths())'` maybe?

Reply:
Or I guess `RUN conda run -n pvs_like_case_study_r Rscript -e 'renv::restore(library=.libPaths())'` -- I didn't know `conda run` was a thing!

Reply:
I was able to not-too-hackily handle this.